docs/src/supported_releases.md

```diff
@@ -183,10 +183,13 @@ Only the last patch release of each branch is supported.
 We offer two types of support:
 
 Technical support
-:   Technical assistance is offered on a best-effort basis for supported
-    releases only. You can request support from the community on the
-    [CloudNativePG Slack](https://cloudnativepg.slack.com/) (in the `#general` channel),
-    or using [GitHub Discussions](https://github.com/cloudnative-pg/cloudnative-pg/discussions).
+:   Technical assistance is provided on a best-effort basis and only for supported releases.
+    You can request help from the community through the
+    [#cloudnativepg-users](https://cloud-native.slack.com/archives/C08MAUJ7NPM) channel
+    on the CNCF Slack workspace (if you're not yet a member, you can join via the
+    [Community Inviter App](https://communityinviter.com/apps/cloud-native/cncf))
+    or through [GitHub Discussions](https://github.com/cloudnative-pg/cloudnative-pg/discussions)
```

Comment:
Here too I'd refer to the Communications section of the README.

Reply:
```suggestion
    You can request help from the community through the CNCF Slack worspace
    or GitHub Discussions.
    We refer you to our [Communication channels](../../README.md#communications)
```

Reply:
I had a similar idea Jaime, but it was primarily influenced by the fact that URLs would regularly change in the past. Now, with the CNCF Slack, these links will become immutable, so I believe it is fine.